scripts/release.py

```diff
@@ -0,0 +1,228 @@
+"""Prepare a Rasa OSS release.
+
+- creates a release branch
+- creates a new changelog section in CHANGELOG.rst based on all collected changes
+- increases the version number
+- pushes the new branch to GitHub
+"""
+import argparse
+import os
+import sys
+from pathlib import Path
+from subprocess import CalledProcessError, check_call, check_output
+from typing import Text, Set
+
+import questionary
+import semantic_version
+from semantic_version import Version
+
+VERSION_FILE_PATH = "rasa/version.py"
+
+REPO_BASE_URL = "https://github.com/RasaHQ/rasa"
+
+RELEASE_BRANCH_PREFIX = "prepare-release-"
+
+
+def create_argument_parser() -> argparse.ArgumentParser:
+    """Parse all the command line arguments for the release script."""
+
+    parser = argparse.ArgumentParser(description="prepare the next library release")
+    parser.add_argument(
+        "--next_version",
+        type=str,
+        help="Either next version number or 'major', 'minor', 'patch'",
+    )
+
+    return parser
+
+
+def project_root() -> Path:
+    """Root directory of the project."""
+    return Path(os.path.dirname(__file__)) / ".."
+
+
+def version_file_path() -> Path:
+    """Path to the python file containing the version number."""
+    return project_root() / VERSION_FILE_PATH
+
+
+def write_version_file(version: Text) -> None:
+    """Dump a new version into the python version file."""
+
+    with version_file_path().open("w") as f:
+        f.write(
+            f"# this file will automatically be changed, \n"
```

Comment:
black complains for me
```suggestion
            f"# this file will automatically be changed,\n"
```